controller/users_blackbox_test.go

```diff
@@ -1430,6 +1432,69 @@ func (s *UsersControllerTestSuite) TestCreateUserAsServiceAccountWithAllFieldsOK
 	require.Equal(s.T(), uint64(1), s.clusterServiceMock.LinkIdentityToClusterCounter)
 }
 
+func (s *UsersControllerTestSuite) TestCreateUserAsServiceAccountWhenFailedForLinkingIdentityToCluster() {
```

Comment:
This is really big name, can we think of some better little small name :-)  Go is not as much as verbose as Java and we should try to align with Go verbosity.  If we see go src code, we generally find small names, we should take inspiration from same.

Reply:
I am fine with it as long as it's readable instead of using short names and getting confused